Src/PRuntimes/PSymRuntime/SymbolicRegressionTests/Integration/Correct/TwoPhaseCommitSync/Client.p

```diff
@@ -84,6 +84,6 @@ In P, function declarations without body are considered as foreign functions.
 fun ChooseTransaction(): tRecord
 {
     return default(tRecord);
-    //return (key = choose(10), val = choose(10));
```

Comment:
Can we create a separate CR for making the P tutorials working with PSym. I am not sure if it is good idea at all. As I do not want to over simplify the tutorials just because we want it to work with PSym.

Reply:
No changes are made to P Tutorial examples in this PR. 
These examples are regression tests for symbolic engine, containing simplified & modified P models originally taken from Tutorial examples.

Will create a separate PSym CI for Tutorial examples.